gulpfile.js

```diff
@@ -1,93 +1,79 @@
-const { src, dest, task, series } = require('gulp');
-const { spawn } = require('child_process');
-const through2 = require('through2');
-const luamin = require('luamin');
-const del = require('del');
 const gulp = require('gulp');
+const del = require('del');
 
-const _compileMoonScript = () => through2.obj((file, _, cb) => {
-	if (file.isBuffer()) {
-		const moonc = spawn('moonc', ['--']);
+const renderSvg = require('./tools/inkscape');
+const minifyLua = require('./tools/luamin');
+const compileMoonscript = require('./tools/moonscript');
 
-		let stdout = '';
-		let stderr = '';
+/**
+ * Cleans the build directories.
+ */
+function clean() {
+	return del(['gamemodes/**/*.lua']);
+}
+clean.description = "Cleans the build directories.";
```

Comment:
Mixed quotes

Reply:
My quotes are mixed just the way I like them :v